llvm/utils/TableGen/ARMTargetDefEmitter.cpp

```diff
@@ -94,19 +94,21 @@ static void EmitARMTargetDef(RecordKeeper &RK, raw_ostream &OS) {
     else
       OS << ", \"" << Alias << "\"";
     OS << ", AArch64::" << AEK;
-    if (AEK == "AEK_NONE") {
+    auto Name = Rec->getValueAsString("Name");
+    if (Name.empty()) {
```

Comment:
Shouldn't we be checking `IsFMVOnly` here?

Reply:
No, for example all the features I have fused  (FEAT_DPB, FEAT_DPB2, FEAT_FLAGM2, FEAT_FRINTTS, FEAT_RCPC2) as well as BTI, are FMVOnly and still have a valid +/- TargetFeatureName.